src/components/SpecialResolution/SpecialResolutionForm.vue

```diff
@@ -0,0 +1,433 @@
+                    :inputRules="signatureDateRules"
+                    @emitDateSync="onSigningDateSync"
+                  />
+                </v-col>
+              </v-row>
+            </v-card>
+
+        </section>
+      </v-card>
+  </v-card>
+</template>
+
+<script lang="ts">
+import { Component, Mixins, Watch } from 'vue-property-decorator'
+import { Action, Getter } from 'vuex-class'
+import { ActionBindingIF, HelpSectionIF, ResourceIF,
+  FormIF, SigningPersonIF, EmptySigningPersonIF, CreateResolutionIF } from '@/interfaces/'
+import { DateMixin } from '@/mixins/'
+import { HelpSection } from '@/components/common/'
+import { DatePicker as DatePickerShared } from '@bcrs-shared-components/date-picker/'
+
+@Component({
+  components: {
+    HelpSection,
+    DatePickerShared
+  }
+})
+export default class SpecialResolutionForm extends Mixins(DateMixin) {
+  @Getter getResource!: ResourceIF
+  @Getter getBusinessFoundingDate!: string
+  @Getter getCurrentDate!: string
+  @Getter getCurrentJsDate!: string
+
+  @Action setResolution!: ActionBindingIF
+  @Getter getcreateResolution!: CreateResolutionIF
+  @Getter getComponentValidate!: boolean
+
+  // Refs
+  $refs!: {
+    resolutionDatePickerRef: DatePickerShared,
+    signatureDatePickerRef: DatePickerShared,
+    signingPersonFormRef: FormIF,
+  }
+
+  // Date properties
+  protected resolutionDateText = ''
+  protected signatureDateText = ''
+
+  readonly MAX_RESOLUTION_TEXT_LENGTH = 1000
+  protected resolutionText = ''
+
+  protected signingPerson: SigningPersonIF = null
+
+  /** Validation rule for individual name fields */
+  readonly firstNameRules = this.nameRules('First Name')
+  readonly middleNameRules = this.nameRules('Middle Name', false)
+  readonly lastNameRules = this.nameRules('Last Name')
+
+  get helpSection (): HelpSectionIF {
+    return this.getResource.changeData?.specialSpecialResolution?.helpSection
+  }
+
+  get getCreateResolutionResource (): any {
+    return this.getResource.changeData?.specialSpecialResolution?.sampleFormSection || {}
+  }
+
+  get documentURL (): string {
+    const docUrl = sessionStorage.getItem('BASE_URL') +
+      this.getCreateResolutionResource?.downloadDocPath
+    return docUrl
+  }
+
+  /** The name section validity state (when prompted by app). */
+  get invalidNameSection (): boolean {
+    // add more state here
+    return (this.getComponentValidate)
+  }
+
+  /** The minimum date that can be entered (can't be earlier than incorporation date ). */
+  get resolutionDateMinStr (): string {
+    return this.dateToYyyyMmDd(this.apiToDate(this.getBusinessFoundingDate))
+  }
+
+  /** The minimum date that can be entered (can't be earlier than incorporation date ). */
+  get resolutionDateMin (): Date {
+    return this.apiToDate(this.getBusinessFoundingDate)
+  }
+  /** The maximum date that can be entered (today). */
+  get resolutionDateMax (): Date {
+    return this.apiToDate(this.getCurrentDate)
```

Comment:
`getCurrentDate` is not in API format. Use a different method to convert yyyy-mm-dd to a Date.

Reply:
Also on lines 341 and 351.